ansible_base/rest_filters/rest_framework/order_backend.py

```diff
@@ -52,6 +52,8 @@ def get_default_ordering(self, view):
         ordering = getattr(view, 'ordering', None)
         if isinstance(ordering, str):
             return (ordering,)
+        elif ordering is None:
+            return ("pk",)
         return ordering
```

Comment:
Can ordering be a non-None string here? (i.e. an int) ... after looking at this I'm realizing it might be a tuple. 
Maybe add a comment like:
```suggestion
        if isinstance(ordering, str):
            # If we got just a string turn it into a tuple of one object
            return (ordering,)
        elif ordering is None:
            # If unspecified, than default to the pk (which all objects should have)
            return ("pk",)
        # If we got something else just return it and let Django error if its not valid
        return ordering
```